src/src/modules/geolocation/geolocation.controller.ts

```diff
@@ -1,17 +1,17 @@
-import { Controller, Get, Param, UseInterceptors } from '@nestjs/common';
-import { GeolocationService } from './geolocation.service';
-import { SentryInterceptor } from 'src/common/interceptors/sentry.interceptor';
+import { Controller, Get, Param, UseInterceptors } from "@nestjs/common";
+import { SentryInterceptor } from "src/common/interceptors/sentry.interceptor";
+import { GeolocationService } from "./geolocation.service";
```

Comment:
why change this sequence.

Reply:
done via command -> typescript -> sort imports